finder/finder.cc

```diff
@@ -123,6 +124,15 @@ long FinderServiceImpl::GetFoodID(const string& food_name) {
   return it->second;
 }
 
+Status FinderServiceImpl::UpdateSupplier(grpc::ServerContext* context,
+                                         const supplyfinder::SupplierInfo* info,
```

Comment:
The fact that the finder talks to the supplier is an implementation detail, and GKE should be able to provide this address directly to your binary on startup.